### DISCUSSION TITLE
Continuing PiPedal develpment in new fork, and call for developers

### DISCUSSION THREAD

**Discussion by FoolHen** [rerdavies/pipedal#147] [General]
Hi, everyone. 

As some of you have noticed, unfortunately @rerdavies has been nowhere to be seen for almost a year. We do not have any information on what the reason might be, so the only thing we can do at this point is for us to pick it up.

I have been using this program for a while, and think is incredibly useful and well done. Because of that I have started to tinker with it to see if some of the issues that I encountered could be fixed. I have created a fork: https://github.com/FoolHen/pipedal and created a new [release v1.1.32](https://github.com/FoolHen/pipedal/releases/tag/v1.1.32) with some fixes and 150 new presets for ToobML, some with knob control support. You can find the changelog in the linked release page.

However, I don't have much experience with c++, especially not with linux and audio processing. My intention is to continue working on it, at least fixing the main issues and polishing it. But I won't be able to do it alone. I'm calling for help, if any of you is interested in helping with continuing the development it would be amazing and greatly appreciated. Please reach out if you are interested😄 

Hope to hear from you guys, and let me know how the new version goes. Feel free to open issues in the fork repo

**Comment by maarthome** [comment]
Hi FoolHen,

Wonderful that you keep the heritage of Robin Davis alive :-)
I do want you to know that I am following everything that happens over here because pipedal is really fantastic!!

Unfortunately I won't be able to contribute much because I'm not a programmer...
Testing however I can do :-)

Thanks and enjoy your weekend!
Maarten

**Comment by EE1977** [comment]
Hi FoolHen!! Great to hear that pipedal is still a live project!! Thank you for all your effort in bringing it live again!! Pipedal is just awesome!! I just have an intermediate experience in Arduino programming and nothing more than that. I can't imagine how I could be of any more help apart from testing. I'll check your new version as soon as possible and I'll report back here, regarding the results. Thank you very very much!!!!

**Comment by 38github** [comment]
Davies is back and hopefully you can contribute and work with Davies.
